### ISSUE TITLE
scoped_id in json formatter

### ISSUE THREAD

**Issue by p-mongo** [rspec/rspec-core#2559]
### Subject of the issue

Is there interest to have scoped_id exposed separately in json formatter? I think it would be helpful.

**Comment by JonRowe** [comment]
Can I ask why you want this? `id` contains the `scoped_id` with the correct file for the example, `scoped_id` on its own is of no use to RSpec (so its sort of private as its only public use is to build `id`)

**Comment by JonRowe** [comment]
To be clear, id is `"#{file_path}:[#{scoped_id}]`, so sorting on it will give the same result.

**Comment by p-mongo** [comment]
Yep thanks. I need to track file paths in addition to scoped ids, using the id field will be less work as it's one field instead of two.
